src/Client.php

```diff
@@ -84,8 +84,23 @@ public function createRequest(Order $order) {
         );
     }
 
-    public function sendOrder(Order $order) {
-        $request = $this->createRequest($order);
+
+	/**
+	 * @param Order $order
+	 * @return string|bool
```

Comment:
Ještě se na to musím podívat pořádně, ale přijde mi že to nikdy nemůže vrátit bool, protože pokud volání selže, tak to vyhodí exception.

Reply:
vzal jsem tu anotaci z https://www.php.net/manual/en/function.curl-exec.php . false je tam ošetřený, akorát sem si nebyl jistý tím true, proto jsem tam nechal i ten bool. Pak jsem si ještě všiml že někde jsou duplicitně anotace s typem a někde je jen typ, tak nevím (ale to už je asi drobnost)

Reply:
@soukicz omlouvám se, ale zjistil jsem, že úpravu vlastně nepotřebuju - teď mi došlo, že si ten request můžu poslat sám např. přes Guzzle (potřeboval jsem to rozdělit kůli logování requestu). Pokud přecejen uznáš, že je úprava užitečná, tak to můžeme dotáhnout. V opačném případě klidně zavři.

Reply:
Není to tohle? https://github.com/soukicz/zbozicz#paraleln%C3%AD-odesl%C3%A1n%C3%AD-objedn%C3%A1vek - jen teda bez paralelního zpracování, ale s posláním přes Guzzle (nebo něco jiného) už to právě počítá.

Reply:
Není to tohle? https://github.com/soukicz/zbozicz#paraleln%C3%AD-odesl%C3%A1n%C3%AD-objedn%C3%A1vek - jen teda bez paralelního zpracování, ale s posláním přes Guzzle (nebo něco jiného) už to právě počítá.

Reply:
jj.. to je přesně ono, tím že jsem zvyklý poslední dobou nepoužívat use, tak jsem přehlídl, že je to GuzzleHttp\Psr7\Request a myslel jsem, že je to nějaký "Soukicz\Zbozicz\Request", takže jsem to slepě upravil. Jelikož se úprava stává neužitečná, tak PR zavírám.